### PR TITLE
Lps 74148 8 7

### PR DESCRIPTION
**Pull request by slnn**

### CODE REVIEW

**Comment by dantewang** [comment]
Fetched

**Comment by slnn** [comment]
Hi @dantewang 

Do we need do more in this pull? Or for other models. do we still need make a check about the sort and group by the order in serivce.xml?

File: modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java [hunk 1]
```diff
@@ -1388,8 +1388,6 @@ public CPDefinitionLocalizationModel newCPDefinitionLocalizationModel(
 
 		//Other fields
```
Comment (dantewang) [inline modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java]: This should only have 2 groups: Localized entity and Auto generated

Comment (dantewang) [reply modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java]: To make it consitent, you should also move `cpDefinitionLocalizationModel.setCompanyId(_companyId);` to `// Auto generated` cuz' it's not defined by service.xml.

File: modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java [hunk 2]
```diff
@@ -1403,6 +1401,11 @@ public CPDefinitionLocalizationModel newCPDefinitionLocalizationModel(
 		cpDefinitionLocalizationModel.setMetaKeywords(
 			"Meta-keywords for definition " + cpDefinitionId);
 
+		//Auto generated
```
Comment (dantewang) [inline modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java:1404]: Remember to add a space between `//` and comment.

**Comment by dantewang** [comment]
@slnn Please see my comment and resend. Please also merge my SF into the changes.

**Pull request closed without merging**